test/run_test.py

```diff
@@ -1715,6 +1727,7 @@ def __str__(self):
     test_batch = TestBatch("tests to run", include, False)
     test_batch_exclude = TestBatch("excluded", exclude, True)
 
+    print_to_stderr(f"Running parallel tests on {NUM_PROCS} processes")
```

Comment:
Is PYTORCH_TEST_RUN_EVERYTHING_IN_SERIAL environment variable still respected? 
I am observing 2 processes now even with 
export PYTORCH_TEST_RUN_EVERYTHING_IN_SERIAL=1

Reply:
it should be.  At that point, you might as well as turn on the memory leak checker too PYTORCH_TEST_CUDA_MEM_LEAK_CHECK

Reply:
Thanks Cat! I was misled by the "Running parallel tests" print and I checked the tests were indeed still serial with that environment variable. 
Considering the case when everything is run in serial, has the amount of testing time increased? I seem to observe some 20% - 30% execution time increase the last month.

Reply:
I don't have the numbers on me right now, but anecdotally TTS has increased from strictly below 3h last year to above 3h currently, and that's with parallelization and sharding.  I can try to get the numbers if you want

Reply:
Thanks Cat. It might just be our internal infra. Let me dig a bit more and will reach out again when I really need that number. Thanks in advance!
Resolving this conversation for now.